TranquilApp/ViewControllers/NotificationViewController.swift

```diff
@@ -22,29 +29,88 @@ class NotificationViewController: UIViewController {
         appearance.shadowColor = .white
         navigationController?.navigationBar.standardAppearance = appearance;
         navigationController?.navigationBar.scrollEdgeAppearance = navigationController?.navigationBar.standardAppearance
-
-        button.setTitle("Press to go from notification to Breathing View", for: .normal)
-        button.titleLabel?.numberOfLines = 0 // 0 indicates dynamic
-        button.titleLabel?.lineBreakMode = NSLineBreakMode.byWordWrapping
-        button.titleLabel?.textAlignment = .center
-        view.addSubview(button)
-        button.backgroundColor = .lightGray
-        button.setTitleColor(.black, for: .normal)
-        button.frame = CGRect(x: 35, y:160, width: 330, height: 600)
-        button.layer.cornerRadius = 10
-        button.addTarget(self, action: #selector(didTapButton), for: .touchUpInside)
         
+        name.textAlignment = .center
+        name.text = "What exercise will help your anxiety right now?"
+        name.textColor = UIColor(SECONDARY_TEXT_COLOR)
+        name.frame = CGRect(x: 25, y: 100, width: 350, height: 70)
+        name.font = .systemFont(ofSize: 25, weight: UIFont.Weight(rawValue: 10))
+        name.isUserInteractionEnabled = false
+        view.addSubview(name)
+        
+        
+        addWaves()
+        addBreathingButton()
+        addJournalButton()
+        addAiChatbotButton()
+        
+    }
+    
+    @objc private func addWaves(){
+        background.view.frame = CGRect(x: 0, y: 0, width: UIScreen.main.bounds.width, height: UIScreen.main.bounds.height)
+        background.didMove(toParent: self)
+        background.modalPresentationStyle = .fullScreen
+        view.insertSubview(background.view, at: 0)
+    }
+    
+    @objc private func addBreathingButton() {
+        breathingButton.setTitle("Press to go from notification to Breathing Exercises", for: .normal)
```

Comment:
Don't think we need the "from notification" part... "Press To Go to Breathing Exercises" seems fine

Reply:
^^ with subsequent buttons as well

Reply:
yeah good idea